packages/jbrowse-web/public/index.html

```diff
@@ -25,7 +25,7 @@
     -->
     <title>JBrowse</title>
   </head>
-  <body>
+  <body style="overscroll-behavior: none">
```

Comment:
this has to be put directly on the body unfortunately.

Reply:
What is the behavior that happens that requires this?

Reply:
on macos, scrolling left and right with the touchpad causes the whole body to "bounce" left and right.  little video about this (in a different project) is https://twitter.com/smashingmag/status/1029709428289937408?lang=en

Reply:
testing out on mac, I can see that this is happening on sidescroll which is undesirable. possibly, we could ignore sidescroll events at the app-level with a event.preventDefault but this may be ok